### PR TITLE
adding Contributing and Code of Conduct pages

### PR DESCRIPTION
**Pull request by Jason9199**

### CODE REVIEW

**Comment by Jason9199** [comment]
Item on Issue #142

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (richardwestenra) [inline code-of-conduct.md]: [INSERT EMAIL ADDRESS] -> together@carpoolvote.com

Comment (Jason9199) [inline code-of-conduct.md]: I'll be happy to make these changes first. Thank you for the approval!

Comment (richardwestenra) [inline code-of-conduct.md]: Sweet as, thanks! Ping me when you've updated the PR.

**Comment by Jason9199** [comment]
Hey @richardwestenra ! The changes you asked for should be ready.

**Comment by richardwestenra** [comment]
hmmm I'm not sure what happened on that master merge to add half the commit history in here. @Jason9199 fid you do some kind of rebase?

This PR looks good but I need to confirm that it's not going to add a bunch of duplicate commits before I merge it to master. @jkbits1 What's your take?

**Comment by Jason9199** [comment]
I just fetched from the master branch to update my own before I committed anything. I didn't use the rebase command.

**Comment by richardwestenra** [comment]
@Jason9199 okay cool, thanks. I'm just doublechecking because usually when I merge master, it doesn't show any commits that already exist on master. Weird :(

**Comment by richardwestenra** [comment]
okay I just did a local test merge and everything looks fine, so I'm gonna go ahead and merge. Thanks for your patience!

**Pull request merged**